logs/Lahi/financial-support/cnfirm.json

```diff
@@ -0,0 +1,193 @@
+{
+    "context": {
+        "domain": "onest:financial-support",
+        "action": "confirm",
+        "version": "1.1.0",
+        "bap_id": "lahi-scholarship-dev-bap-network.tekdinext.com",
+        "bap_uri": "https://lahi-scholarship-dev-bap-network.tekdinext.com",
+        "bpp_id": "d.tibil.scholarship.dash.tibilprojects.com",
+        "bpp_uri": "https://d.tibil.scholarship.dash.tibilprojects.com/protocol-network",
+        "transaction_id": "619ea787-6d74-4c5e-943d-68aeeeed41d0",
+        "message_id": "42bccc28-ed19-414f-86e6-948aed7b0807",
+        "timestamp": "2024-05-23T13:47:04.273Z"
+    },
+    "message": {
+        "order": {
+            "provider": {
+                "id": "1",
+                "descriptor": {
+                    "name": "buddy4"
+                }
+            },
+            "items": [
+                {
+                    "id": "1",
+                    "descriptor": {
```

Comment:
Confirm payload is incorrect, please check the implementation guide and fix it.
https://github.com/ONEST-Network/ONEST-Specification/blob/main/docs/implementation_guides/financial-support/implementation_guide.md